KP5-Topic2/KP4-Topic2/target/m2e-wtp/web-resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files